docs/get-started/running-nodes.mdx

```diff
@@ -4,306 +4,218 @@ title: 'Running Etherlink Nodes'
+The Smart Rollup node verifies the integrity of the kernel based on a hash posted onchain during the origination of Etherlink.
+
+To start, the node uses the _installer kernel_, which is a compressed version of the kernel that provides only enough information to install the original kernel.
+The data for the original kernel is stored in separate files called _preimages_.
+For more information about installer kernels and preimages, see the tutorial [Deploy a Smart Rollup](https://docs.tezos.com/tutorials/smart-rollup) on docs.tezos.com.
+
+The Etherlink team maintains a file server with the preimages on a so-called "preimages endpoint".
+These instructions cover how to start the Smart Rollup node using these preimages.
+To build them yourself, see [Building the Etherlink kernel](#building-the-etherlink-kernel).
+
+For simplicity, these steps show how to run the node in observer mode.
+
+1. Get a built version of the Smart Rollup node binary, named `octez-smart-rollup-node`, in one of these ways:
+
+   - To set up a Smart Rollup node from Etherlink genesis, it's best to use recent Octez source code.
+   One commit that is known to work well is [`c5ed8f9`](https://gitlab.com/tezos/tezos/-/commit/c5ed8f9d5737cec03b29e5a7c3bf2169cc239ea8) from the `master` branch of the [Octez repository](https://gitlab.com/tezos/tezos).
+   For more information about building Octez binaries, see the Octez documentation: https://tezos.gitlab.io/index.html
+
+   - You can also use Octez binary builds.
+   Builds for Linux systems are available for the [x86_64](https://gitlab.com/tezos/tezos/-/jobs/6849203975/artifacts/browse/octez-binaries/x86_64/) architecture.
+
+1. Initialize the local context of the node, which is where it stores local data:
+
+   1. Set the environment variable `sr_observer_data_dir` to the directory where the node should store its local data.
+   1. Initialize the local context by running this command:
+
+      ```bash
+      octez-smart-rollup-node init observer config for sr1Ghq66tYK9y3r8CC1Tf8i8m5nxh8nTvZEf  \
+        with operators --data-dir $sr_observer_data_dir
+      ```
+
+      This command generates a minimal configuration file (`config.json`) in the local data folder:
+
+      ```json
+      { "smart-rollup-address": "sr1Ghq66tYK9y3r8CC1Tf8i8m5nxh8nTvZEf",
+        "smart-rollup-node-operator": {}, "fee-parameters": {},
+        "mode": "observer" }
+      ```
+
+1. Activate the `unsafe-pvm-patches` setting, which is required for Etherlink, and add the location of the preimages by adding these settings to the `config.json` file, as in this example:
+
+   ```json
+   { "smart-rollup-address": "sr1Ghq66tYK9y3r8CC1Tf8i8m5nxh8nTvZEf",
+     "smart-rollup-node-operator": {}, "fee-parameters": {},
+     "unsafe-pvm-patches": [
+       { "increase_max_nb_tick": "50_000_000_000_000" }
+     ],
+     "pre-images-endpoint": "https://snapshots.eu.tzinit.org/etherlink-mainnet/wasm_2_0_0",
```

Comment:
there is a command line argument for setting this normally with  the `init observer config` command

Reply:
Added that in 626f8a4.